backend/app/services/chat/session_manager.py

```
@@ -501,6 +505,80 @@ async def subscribe_streaming_channel(self, subtask_id: int):
+            username: Username of the user
+
+        Returns:
+            bool: True if set was successful
+        """
+        try:
+            key = self._get_task_streaming_key(task_id)
+            value = {
+                "subtask_id": subtask_id,
+                "user_id": user_id,
+                "username": username,
+                "started_at": datetime.now().isoformat(),
```

Ruff (0.14.8)

Undefined name `datetime`

(F821)